src/__tests__/index.node.js

```diff
@@ -1,42 +1,53 @@
 /* eslint-env node */
-import {fork} from 'child_process';
+import App from 'fusion-core';
 import test from 'tape-cup';
-import ErrorHandling from '../server';
+import {getSimulator} from 'fusion-test-utils';
+import {fork} from 'child_process';
+import ErrorHandling, {ErrorHandlerToken} from '../server';
 
 test('request errors', async t => {
+  const app = new App('test', el => el);
+
   let called = 0;
   const onError = () => {
     called++;
   };
-  const middleware = ErrorHandling({onError});
-  await middleware({}, () => Promise.reject(new Error('server error'))).catch(
-    () => {}
-  );
-  t.equals(called, 1, 'emits server error');
+  app.register(ErrorHandling);
+  app.register(ErrorHandlerToken, onError);
+  app.middleware(() => Promise.reject());
 
-  const ctx = {
-    path: '/_errors',
-    prefix: '',
-    request: {body: {message: 'test'}},
-  };
-  await middleware(ctx, () => Promise.resolve());
-  t.equals(called, 2, 'emits browser error');
-  t.end();
+  await getSimulator(app)
+    .request('/_errors', {
+      prefix: '',
+      request: {body: {message: 'test'}},
+    })
+    .catch(() => {});
```

Comment:
I feel like this will make errors hard to discover, since they will be caught and thrown away. I would rather just have an unhandled rejection here